HousingSearchApi/V1/Boundary/Requests/GetAccountListRequest.cs

```diff
@@ -2,22 +2,6 @@ namespace HousingSearchApi.V1.Boundary.Requests
 {
     public class GetAccountListRequest : HousingSearchRequest
     {
```

Comment:
Is this class no longer needed?

Reply:
in the new changes i will put the new fields here.
because of small pr rule, i will do it in the next pr.